src/js/checklist.js

```diff
@@ -58,10 +58,45 @@ function populateChecklistFromLocalStorage(checkboxSelector) {
 
 function processChecklist() {
 	var checkboxSelector = '.c-checklist__checkbox input[type="checkbox"]';
-
 	populateChecklistFromLocalStorage(checkboxSelector);
 	processChecklistClick(checkboxSelector);
 }
 
 openLinkedCheckListItem();
 processChecklist();
+
+function registerToggleButton(buttonEl, parentEl) {
+	if (buttonEl && parentEl) {
+		var details = parentEl.querySelectorAll("details");
```

Comment:
Can we make this selector a little more specific? Like `details.c-checklist` to ensure that we are not toggle any other `<details>` that might be there. Just to future-proof it a bit.